src/reanimated2/layoutReanimation/web/transition/Sequenced.web.ts

```diff
@@ -0,0 +1,42 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+import type { TransitionData } from '../animationParser';
+
+export function SequencedTransition(
+  name: string,
+  transitionData: TransitionData
+) {
+  const { translateX, translateY, scaleX, scaleY, reversed } = transitionData;
+
+  const scaleValue = reversed ? `1, ${scaleX}` : `${scaleY}, 1`;
+
+  // TODO: Change proportions
```

Comment:
TODO

Reply:
Removed in [95cb5ca](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/95cb5ca6b2338345dddd1d28fed7d2f1e1397dc4).